.mci.yml

```diff
@@ -617,7 +617,7 @@ buildvariants:
     #         Linux Buildvariants         #
     #######################################
     - name: ubuntu1604-release
-      display_name: "Ubuntu 16.04 Release"
+      display_name: "Ubuntu 16.04 Release (MongoDB Latest)"
```

Comment:
For convenience. Each variant specifies the server version as a variant expansion, each variant runs exactly one server version.

Reply:
Acknowledged.